Source/Core/Core/State.cpp

```diff
@@ -391,6 +399,11 @@ static void CompressAndDumpState(CompressAndDumpState_args save_args)
 
 void SaveAs(const std::string& filename, bool wait)
 {
+	if (NetPlay::IsNetPlayRunning())
+	{
+		OSD::AddMessage("Access to savestates disabled in Netplay to prevent desyncs");
+		return;
+	}
```

Comment:
Is this necessary? Saving a state shouldn't affect sync, otherwise I think something is very broken for TASing. Being able to save a state while using netplay might be useful for going back to a situation in a game later when not using netplay.